packages/parsec-deb/parsec-deb.pacscript

```diff
@@ -0,0 +1,6 @@
+name="parsec-deb"
+gives="parsec"
+version="150-86e"
+url="https://builds.parsec.app/package/parsec-linux.deb"
+description="Simple, low-latency game streaming"
+hash="c30072f3a4ddac7687f626b8d32876e327791bce164d74448a147ef48ea8eae5"
```

Comment:
As the `url` is a dynamic link, the `hash` would be automatically invalidated in the future when a new version releases. Consider using a static `url` instead (an `url` where you can plug the `version` in), or remove the `hash`.

---

Add `maintainer`.

Reply:
They don't offer versioned URLs. I'm also not going to maintain this.

Reply:
```suggestion
version="150-86e"
url="https://builds.parsec.app/package/parsec-linux.deb"
description="Simple, low-latency game streaming"
```

Remove `hash` then